src/app/pages/home/footer/footer.component.html

```diff
@@ -1 +1,22 @@
-<div>Footer Page</div>
\ No newline at end of file
+
+<nav class=" navbar navbar-dark bg-dark mt-10 clearfix">
+
+    <div class="navbar-expand m-auto navbar-text">
+        Made 
+        <i class="fas fa-heart">
+
+        </i>
+        by NTUOSS
+        
+    </div>    
+    <div class="navbar-expand m-auto navbar-text ">
+        Follow Us On Social Media
+        <a href= "https://www.facebook.com/ntuoss/?ref=br_rs">
+        <button type = "button" class = "btn">

```

Comment:
It will be better to use the font-awesome icon for Facebook, Instagram, Github, and stuff.

Reply:
https://github.com/orgs/ntuoss/projects/1
Also, refer to the project page for more information.